test/unit/config/suppress-browser-warnings.js

```diff
@@ -1,33 +1,32 @@
 /**
  * Suppress specific browser warnings for unit tests
  *
- * This file mocks the @wordpress/warning module to suppress specific
- * warnings that are expected or not relevant in the test environment,
- * while allowing other warnings to pass through normally.
+ * This file mocks the @wordpress/deprecated module to suppress specific
+ * deprecation warnings that are expected or not relevant in the test environment,
+ * while allowing other deprecation warnings to pass through normally.
  *
  * Note: This mock can be removed once the apiVersion:3 becomes the default.
  */
 
-jest.mock( '@wordpress/warning', () => {
-	const mockOriginalWarning =
-		jest.requireActual( '@wordpress/warning' ).default;
+jest.mock( '@wordpress/deprecated', () => {
+	const mockOriginalDeprecated = jest.requireActual(
+		'@wordpress/deprecated'
+	).default;
 
 	return {
 		__esModule: true,
-		default: jest.fn( ( message ) => {
-			const suppressedWarningRegexes = [
-				/The block ".*" is registered with API version 2 or lower/, // apiVersion warnings
+		default: jest.fn( ( feature, options = {} ) => {
+			const suppressedFeatures = [
+				'Block with API version 2 or lower', // apiVersion deprecation warnings
 			];
-			if ( typeof message === 'string' ) {
-				const shouldSuppress = suppressedWarningRegexes.some(
-					( regex ) => regex.test( message )
-				);
-				if ( shouldSuppress ) {
+			if ( typeof feature === 'string' ) {
+				const shouldSuppress = suppressedFeatures.includes( feature );
+				if ( shouldSuppress && options.since === '6.9' ) {
```

Comment:
I'm not sure I understand what the role of the `since: '6.9'` check here is? Can you elaborate? What difference does it make if we remove it?